content/fr/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Web UI (Dashboard)
+content_template: templates/concept
+weight: 10
+card:
+  name: tasks
+  weight: 30
+  title: Utiliser le tableau de bord Web UI
+---
+
+{{% capture overview %}}
+
+Dashboard UI est une interface web pour Kubernetes.
+Vous pouvez utiliser Dashboard pour déployer des applications conteneurisées sur un cluster Kubernetes, dépanner votre application conteneurisée et gérer les ressources du cluster.
+Vous pouvez utiliser Dashboard pour obtenir une vue d'ensemble des applications en cours d'exécution sur votre cluster, ainsi que pour créer ou modifier des ressources Kubernetes individuelles. (comme des Deployments, Jobs, DaemonSets, etc).
+Par exemple, vous pouvez redimensionner un Deployment, lancer une mise à jour progressive, redémarrez un pod ou déployez de nouvelles applications à l'aide d'un assistant de déploiement.
+
+Le tableau de bord fournit également des informations sur l'état des ressources Kubernetes de votre cluster et sur les erreurs éventuelles.
+
+![Kubernetes Dashboard UI](/images/docs/ui-dashboard.png)
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Déploiement du Dashboard UI
+
+L'interface utilisateur du tableau de bord n'est pas déployée par défaut.
+Pour le déployer, exécutez la commande suivante:
+
+```text
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-beta6/aio/deploy/recommended.yaml
+```
+
+## Accessing the Dashboard UI
+
+Pour protéger vos données de cluster, Dashboard se déploie avec une configuration RBAC minimale par défaut.
```

Comment:
Pour protéger vos données dans le cluster, le tableau de bord se déploie avec une configuration RBAC minimale par défaut.

Reply:
Maybe use a glossary tooltip for RBAC (it's jargon, but it's jargon that abbreviates English).